lib/resources/collection/index.js

```diff
@@ -589,14 +597,20 @@ Collection.prototype.save = function (ctx, fn) {
       if(session && session.emitToAll) session.emitToAll(collection.name + ':changed');
     }
   }
+  
+  function post() {
```

Comment:
Could you find a better name? (preparePost? validatePost?)
Between doPost() and post(), I don't know which one is the one to use